tensorrt_llm/serve/tool_parser/glm47_parser.py

```
@@ -0,0 +1,480 @@
+                    continue
+                func_name = func_detail.group(1).strip() if func_detail.group(1) else ""
+                func_args = func_detail.group(2) if func_detail.group(2) else ""
+                arguments = {}
+                if func_args:
+                    pairs = self.func_arg_regex.findall(func_args)
+                    arguments = self._parse_argument_pairs(pairs, func_name, tools)
+
+                match_result = {"name": func_name, "parameters": arguments}
+                calls.extend(self.parse_base_json(match_result, tools))
+            return StreamingParseResult(normal_text=normal_text, calls=calls)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)